### PR TITLE
Fixes several syntax and template issues with LDAP resources.

### PR DESCRIPTION
**Pull request by johnbellone**
### Description

There were several issues with how we are setting
variables (e.g. using << operand vs assignment) and places where we
should be printing the array instead of a string.

### Issues Resolved

[List any existing issues this PR resolves]

### Check List
- [ ] All tests pass. See https://github.com/sous-chefs/grafana/blob/master/TESTING.md
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This Pull Request is probably missing tests.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ This file is used to list changes made in each version of grafana.
 
 ## Unreleased
```
Comment (xorima) [inline CHANGELOG.md]: Can you flip this to 4.0.2 and set today's date please, then we can get this merged and released

**Comment by xorima** [comment]
Really nice work, thank you for this, 
Once the small change has been done, we can get this released to supermarket

**Review by xorima — APPROVED** [review]

**Comment by xorima** [comment]
Released to supermarket, thanks for this :)

**Pull request merged**